ui/00_message/common/order_condition.xml

```diff
@@ -89,5 +91,6 @@
   <message>
     <key>ORDER_CONDITION_1014</key>
     <original>パーティプレイ不可（メインポーン&lt;QCND 0&gt;人同行可）</original>
+    <translation>No party members (&lt;QCND 0&gt; Pawn allowed)</translation>
```

Comment:
Yeah was wondering about that, I assumed the qcnd was code that fetched if it was main or support pawn. 

Ill translate it to Main QCND pawn allowed, or perhaps QCND Main pawn allowed.

I think a description of what those codes do would be helpful to know where to put them.